csxtools/utils.py

```diff
@@ -128,6 +127,93 @@ def get_fastccd_images(light_header, dark_headers=None,
     return _correct_fccd_images(events, bgnd, flat, gain)
 
 
+def get_axis_images(light_header, dark_header=None, flat=None, tag=None, roi=None):
+    """Retreive and correct AXIS1 Images from associated headers
+
+    Retrieve AXIS1 Images from databroker and correct for:
+
+    -   Bad Pixels (converted to ``np.nan``)
+    -   Backgorund.
+    -   Flatfield correction.
+    -   Rotation (returned images are rotated 90 deg cw)
+
+    Parameters
+    ----------
+    light_header : databorker header
+        This header defines the images to convert
+
+    dark_headers : databroker headers , optional
```

Comment:
make singular, not plural since only 1 dark header is requires for the axis40 detector